client/src/main/com/sinch/sdk/domains/voice/adapters/CalloutsService.java

```diff
@@ -23,6 +26,21 @@ protected CalloutsApi getApi() {
     return this.api;
   }
 
+  public String textToSpeech(CalloutRequestParametersTTS parameters) {
+
+    return call(parameters);
+  }
+
+  public String conference(CalloutRequestParametersConference parameters) {
+
+    return call(parameters);
+  }
+
+  public String custom(CalloutRequestParametersCustom parameters) {
+
+    return call(parameters);
+  }
+
   public String call(CalloutRequestParameters parameters) {
```

Comment:
Do you still want to expose the generic function?

Reply:
Yes: My thought was to let it public so if final end user choose to store the parameters as `CalloutRequestParameters` "base class" he can call this function.
Following use case may not have sense, to be confirmed, but technically a user could prepare a list of calls and store parameters within a `Collection<CalloutRequestParameters>`
So by batch, or any use case, it could call `call(CalloutRequestParameters parameters)`and not any of dedicated functions